### PR TITLE
Survey page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done
- Updates form elements present on the survey page
- Makes page responsive

## Issue
https://github.com/canonical/web-squad/issues/5687

### CODE REVIEW

**Comment by anthonydillon** [comment]
@petesfrench should this PR be in New issues? Is there an issue for this?

File: indico_themes_canonical/static/css/ubuntu-summit/index.css [hunk 1]
```diff
@@ -35,7 +36,8 @@ TEMP STYLES
 */
 .confSubTitleBox,
 .support_box,
-#session-bar {
+.event-page-header,
+div.footer {
```
Comment (carkod) [inline indico_themes_canonical/static/css/ubuntu-summit/index.css:40]: Do you need this specificity? Doesn't it work with simply `.footer`?

**Review by carkod — APPROVED** [review]
LGTM, but maybe it needs a bit of design? It doesn't use Vanilla?

**Comment by petesfrench** [comment]
I am addressing design changes in [another PR](https://app.zenhub.com/workspaces/-sites-tribe-5931746dba512f05402b61f6/issues/canonical/web-squad/5903). As not to block the other PR's from being reviewed.

**Pull request merged**